.github/workflows/ci.yml

```diff
@@ -41,6 +42,6 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         MIX_ENV: test
-        # AUTH_API_KEY: 2PzB7PPnpuLsbWmWtXpGyI+kfSQSQ1zUW2Atz/+8PdZuSEJzHgzGnJWV35nTKRwx
+        AUTH_API_KEY: ${{ secrets.AUTH_API_KEY }}
```

Comment:
The `test` version of the `AUTH_API_KEY` could be _fake_ and `public` because there shouldn't be an actual request made ... 💭 (_but I totally get the that you're making it secret so that others don't make the mistake of making it `public` ..._) 👍